### DISCUSSION TITLE
chance of different background for "song playing" pop up

### DISCUSSION THREAD

**Discussion by Ronikusu** [ppy/osu#24325] [Q&A] [answered]
there is a small chance of a different background of a song downloaded showing up instead of the original background for the "song playing" pop up.

![Screenshot 2023-07-21 161634](https://github.com/ppy/osu/assets/124932615/d8cce09f-9cf1-4035-9094-c7ef755fe004)

where the background came from:

![image](https://github.com/ppy/osu/assets/124932615/e263c004-9631-44ef-a6c6-52243b223978)

**Comment by peppy** [comment]
Can you reproduce this often? Do you remember if when this happened, if you went back to the last track would it fix the background?

Also, does "baby cruising love" show the correct background at song select?

**Reply by Ronikusu** [reply]
you can produce this often.
if I go back to the last track, it does fix background.
it does show the correct background at song select.
sorry for the late response, I do not use Github much.

**Reply by peppy** [reply]
Does it happen when going to previous tracks or next tracks or both?

Any further details you can provide would be appreciated.

**Reply by Ronikusu** [reply]
Just tested it out for previous tracks and it will do this (but it does take longer to reproduce)

**Reply by peppy** [reply, marked as answer]
Should be addressed via https://github.com/ppy/osu/pull/24418.
